cuda_pathfinder/docs/source/index.rst

```diff
@@ -0,0 +1,24 @@
+.. SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+``cuda.pathfinder``: Utilities for locating CUDA components
+===========================================================
+
+.. include:: ../../cuda/pathfinder/README.md
+   :parser: myst_parser.sphinx_
```

Comment:
Leaving a note here, this requires Ralf's attention.

Reply:
@leofang As far as I can see the rendered pages look good. Is there still anything I should do here?